src/imitation/util/sacred.py

```diff
@@ -1,9 +1,13 @@
 import json
 import os
+import pathlib
 import warnings
 from typing import Any, Callable, List, NamedTuple, Union
 
-import sacred
```

Comment:
Why is `sacred_run` better than `sacred.run`?

Reply:
Hmm looks like I actually misread Google Style guide -- switching this back